src/Api/Api.php

```diff
@@ -234,7 +234,9 @@ public function collection(
 		array|BaseCollection|null $collection = null
 	): Collection {
 		if (isset($this->collections[$name]) === false) {
-			throw new NotFoundException(sprintf('The collection "%s" does not exist', $name));
+			throw new NotFoundException(
+				message: sprintf('The collection "%s" does not exist', $name)
```

Comment:
Should we do this in all cases where we only pass a message. I've seen various refactored Exception calls with messages in previous files, where you only changed the indentation. I personally really like the more explicit version with the named argument.